packages/client/hmi-client/src/components/data-explorer/search-item.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts">
+import { XDDArticle } from '@/types/XDD';
+import { isXDDArticle } from '@/utils/data-util';
+import IconAdd24 from '@carbon/icons-vue/es/add/24';
+import IconCheckmark24 from '@carbon/icons-vue/es/checkmark/24';
+import { ResultType } from '@/types/common';
+
+const props = defineProps<{
+	d: XDDArticle;
+	previewedArticle: XDDArticle | null;
+	selectedSearchItems: ResultType[];
+}>();
+
+const emit = defineEmits(['toggle-article-selected', 'toggle-article-preview']);
+
+const isSelected = () =>
+	props.selectedSearchItems.find((item) => {
+		if (isXDDArticle(item)) {
+			const itemAsArticle = item as XDDArticle;
+			return itemAsArticle.title === props.d.title;
+		}
+		return false;
+	});
+
+const formatTitle = () => (props.d.title ? props.d.title : props.d.title);
```

Comment:
What is this?

Reply:
I'll get rid of it - blindly copy pasted that from the original component haha.